src/styles/components/_footer.scss

```diff
@@ -0,0 +1,98 @@
+.default-footer {
+  background-color: black;
+  position: absolute;
+  width: 100%;
+  bottom: 0;
+
+  // Content wrapper
+  &__content {
+    min-height: 332px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    gap: 0.5rem;
+    text-align: center;
+    padding: 3.75rem; /* 60px/16px */
+    font-size: 1.25rem; /* 20px/16px */
+    margin: auto;
+    line-height: 2rem; /* 17px/16px */
+    transition: 500ms color;
+  }
+
+  // Flex items of content wrapper
+  &__item {
+    color: white;
+    width: 100%;
+
+    & p {
+      margin: 0;
+      padding: 0;
+    }
+  }
+
+  &__item a {
+    text-decoration: underline;
+    background: linear-gradient(to left, rgb(0 0 0 / 0%) 50%, gray 50%);
+    background-size: 201% 100%;
+    background-position: right bottom;
+    transition: ease-in-out background-position 250ms;
+  }
+
+  &__item > a:hover,
+  &__item > a:focus-within {
+    background-position: left bottom;
+  }
+
+  @media (prefers-reduced-motion) {
+    &__item a {
+      transition: ease-in-out background-color 250ms;
+    }
+
+    &__item > a:hover,
+    &__item > a:focus-within {
+      background-position: right bottom;
+      background-color: gray;
+    }
+  }
+
+  // Container for icons that link to other websites
+  &__links {
+    margin: 1.5rem 0;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: 22px;
+
+    // The linter doesn't take nesting into account; it's fine this way.
+    /* stylelint-disable-next-line no-descending-specificity */
```

Comment:
I ended up refactoring a little bit to avoid this error instead. Got some advice about dealing with this particular rule from Ben and Catherine--it doesn't look at the DOM and will therefore say that a rule is less specific than another, even if the html tags using that class are exclusively nested inside tags with the more specific rule. It's *generally* useful, so the one-line ignore is often the way to go if the 'fix' is less comprehensible than the original.